uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Hook/PageRun/RelationshipTypesFilter.php

```diff
@@ -13,9 +13,18 @@ public function handle($page) {
     }
 
     $rows = $page->get_template_vars('rows');
+
+    $targetRelTypes = [
+      'Case Coordinator is',
+      'Employee of',
+      'Head of Household for',
+      'Household member of',
+    ];
     // remove disabled relationship types
     foreach ($rows as $index => $row) {
-      if (CRM_Utils_Array::value('is_active', $row) == '0'&& in_array(CRM_Utils_Array::value('name', $row),['Case Coordinator is','Employee of','Head of Household for','Household member of'])) {
+      $isActive = CRM_Utils_Array::value('is_active', $row) == '0';
```

Comment:
This variable name doesn't make sense, when I made the suggestions you would also need to change some of the logic

`$isActive = CRM_Utils_Array::value('is_active', $row) == '0';`

If `is_active` is 0 then it's not active, that's why I suggested changing this to `== '1'` and inverting the logic later

Reply:
Ok, changed to '1' and changed the condition